### ISSUE TITLE
Epic: second data entry

### ISSUE THREAD

**Issue by praseodym** [kiesraad/abacus#129]
Second data entry of the full data entry form.

After completing the first data entry, the polling station is open for a second data entry. If the first and second data entry are completely identical, the data entry becomes the definitive polling station results. We'll need to add a `finalised` boolean in the `polling_station_data_entries` to keep track of when the first and second data entries are completed.

Design in [Figma section "Tweede Invoerder"](https://www.figma.com/design/zZlFr8tYiRyp4I26sh6eqp/Kiesraad---Abacus-optelsoftware?node-id=135-2256&t=NhcNn48G4xZpq3jw-4).

TODO:
- add second data entry status to backend, keeping finalise as-is to not break current frontend
- implement second data entry in frontend
- implement finalise to result only when second data entry is equal to first data entry
- second data entry status in coordinator election status screen
- specific feedback on polling station choice page (#388)

Not in scope:
- resolving differences by the coordinator (#130)
- showing warnings to typist about differences with first data entry (#130)
- enforcing that second data entry is done by a different user than the first data entry (draft epic)

**Comment by jorisleker** [comment]
is [this additional validation rule](https://github.com/kiesraad/abacus/blob/main/documentatie/use-cases/GSB-validatieregels-plausibiliteitschecks.md#bij-tweede-invoer-alle-ingevoerde-waardes-van-de-tweede-invoer-zijn-gelijk-aan-die-van-de-eerste-invoer) (second user's input does not match first user's input) in scope?

(we should consider numbering this validation rule, currently it does not have a reference)

**Comment by praseodym** [comment]
> is [this additional validation rule](https://github.com/kiesraad/abacus/blob/main/documentatie/use-cases/GSB-validatieregels-plausibiliteitschecks.md#bij-tweede-invoer-alle-ingevoerde-waardes-van-de-tweede-invoer-zijn-gelijk-aan-die-van-de-eerste-invoer) (second user's input does not match first user's input) in scope?
> 
> (we should consider numbering this validation rule, currently it does not have a reference)

I don't think we have decided this yet, see [this conversation in Figma](https://github.com/kiesraad/abacus/issues/129#issuecomment-2407202991).

**Comment by jorisleker** [comment]
@ring-ring-ring can you help us out here?
